File: Super/Super.java
class Animal {
    Animal(String name) {
        System.out.println("Created " + name);
    }

    void eat() {
        System.out.println("Animal eats");
    }
}

class Dog extends Animal {
    Dog(String name) {
        super(name);
    }

    @Override
    void eat() {
        super.eat();
        System.out.println("Dog chews");
    }
}

public class Super {
    public static void main(String[] args) {
        new Dog("Bruno").eat();
    }
}

File: Super/Super.py
class Animal:
    def __init__(self, name: str) -> None:
        self.name = name

    def eat(self) -> None:
        print(f"{self.name} eats")


class Dog(Animal):
    def eat(self) -> None:
        super().eat()
        print(f"{self.name} chews")


def main() -> None:
    Dog("Bruno").eat()


if __name__ == "__main__":
    main()
